mozregression/launchers.py

```diff
@@ -482,7 +482,14 @@ def _start(
             self.adb.rm(self.remote_profile, recursive=True)
         LOG.debug("Pushing profile to device (%s -> %s)" % (profile.profile, self.remote_profile))
         self.adb.push(profile.profile, self.remote_profile)
-        self._launch()
+        if cmdargs and len(cmdargs) == 1 and not cmdargs[0].startswith("-"):
+            url = cmdargs[0]
+        else:
+            url = None
+        if isinstance(self, (FenixLauncher, FennecLauncher, FocusLauncher)):
+            self._launch(url)
```

Comment:
We should future-proof this by making it `self._launch(url=url)`, in case a positional argument is added in the future.